src/displayapp/screens/Navigation.cpp

```diff
@@ -146,7 +146,7 @@ Navigation::Navigation(Pinetime::Applications::DisplayApp* app, Pinetime::Contro
 
   txtManDist = lv_label_create(lv_scr_act(), nullptr);
   lv_label_set_long_mode(txtManDist, LV_LABEL_LONG_BREAK);
-  lv_obj_set_style_local_text_color(txtManDist, LV_LABEL_PART_MAIN, LV_STATE_DEFAULT, LV_COLOR_GREEN);
+  lv_obj_set_style_local_text_color(txtManDist, LV_LABEL_PART_MAIN, LV_STATE_DEFAULT, LV_COLOR_MAKE(0x00, 0xc0, 0x00));
```

Comment:
why define LV_PINTEIMTE_COLOR_GREEN and then use `LV_COLOR_MAKE` in the code? I don't understand

Reply:
None of the other theme defined colors are used in code either. I don't think `lv_pinetime_theme.h` is the best place to keep our colors, so I didn't want to use that. Ideally we would have the colors in some cpp format, but then they can't be included in the C theme file. I thought this issue should be resolved separately.